src/pages/[platform]/deploy-and-host/fullstack-branching/mono-and-multi-repos/index.mdx

```diff
@@ -34,7 +34,7 @@ You might have different frontend and backend teams that maintain their own repo
 
 ## Deploy the backend app
 
-1. Run `mkdir backend-app && npm create amplify@beta` to set up a backend-only Amplify project. Commit the code to a Git provider of your choice.
+1. Run `mkdir backend-app && cd backend-app && npm create amplify@beta` to set up a backend-only Amplify project. Commit the code to a Git provider of your choice.
```

Comment:
Does this even make sense? We prompt you where you want to create the project, so we could just drop `mkdir backend-app` entirely:

```suggestion
1. Run `npm create amplify@beta` to set up a backend-only Amplify project. Commit the code to a Git provider of your choice.
```

Reply:
![CleanShot 2024-04-17 at 15 20 23](https://github.com/aws-amplify/docs/assets/6165315/07d8aacc-2897-4a13-9514-11f11b394baa)